frontend/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
+    "singleQuote": true
```

Comment:
I added these settings, but am open to inverting/adjusting if there are any strong feelings...